### DISCUSSION TITLE
Partial names dont seem to match `include` names in templates

### DISCUSSION THREAD

**Discussion by ko185061** [linkerd/linkerd2#9797] [Help]
When trying to apply the helm charts via controller for stable-2.12.2 is there a reason the partial template _affinity.tpl is defined as linkerd.pod-affinity but in /charts/linkerd-control-plane/templates/proxy-injector.yaml it calls to include linkerd.affinity? This is causing some issues with the render since it is saying linkerd.affinity does not exist, this is the case for a few other partials as well so want to clarify if Im just misunderstanding

**Comment by alpeb** [comment]
`linkerd.affinity` is a partial defined last in that same `_affinity.tpl` file. Can you provide more details on your setup, or better yet, a minimal repro to your issue?
